main/acle.md

```diff
@@ -2699,11 +2704,34 @@ following:
   in one of the translation units.
   * Implicitly, without this attribute,
   * or explicitly providing the `default` in the attribute.
-* All instances of the versions shall share the same function
-  signature and calling convention.
+* The default version signature is the signature for calling
+  the multiversioned functions.
+* Non-default versions shall share the same calling convention
+  of the default version and have a type that is convertible to the type
+  of the default version.
+
 * All the function versions must be declared at the translation
   unit in which the definition of the default version resides.
 
+For example, the below is valid and 2 is passed as the
+value for `c` when calling `f`.
+
+```c
```

Comment:
NIT: Not sure if it matters, but I think default initialization of arguments is only valid in C++ code, so perhaps the snipped should be marked accordingly?

Reply:
👍 Done

Reply:
I think the markdown annotation for C++ is cpp. You can use build_with_docker.sh to generate locally a pdf version of the document. This way you can double check what the right syntax is.